### PR TITLE
Replace Coil with Glide.

### PR DESCRIPTION
**Pull request by arriolac**

### CODE REVIEW

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt [hunk 1]
```diff
@@ -271,35 +285,56 @@ private fun PlantDetailsContent(
...
-    if (painter.state is AsyncImagePainter.State.Loading) {
-        Box(
+            .height(imageHeight)) {
+        if (isLoading) {
+            Box(
+                Modifier
+                    .fillMaxSize()
+                    .background(placeholderColor)
+            )
+        }
```
Comment (bentrengrove) [inline app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt:307]: Glide doesn't have some sort of placeholder functionality built in?

**Review by bentrengrove — APPROVED** [review]

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt (hunk 1 above)
Comment (arriolac) [reply app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt:307]: Ah - that just got added recently so will update this PR

Comment (arriolac) [reply app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt:307]: Looks like it's not released yet. Will add a comment to update it when that feature is available on maven.

Comment (bentrengrove) [reply app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt:307]: Sounds good!

**Pull request merged**